### ISSUE TITLE
EdgeTPU Compilation Fails for Unknown Reasons Using TFLite Lane Detection Model

### ISSUE THREAD

**Issue by ghost** [google-coral/edgetpu#612]
### Description

EdgeTPU fails compilation on tflite model quantized from float32 to int8 for Coral USB Accelerator requirements.

There is no clear indicator why this is failing currently. Our model was a pytorch-trained model converted to onnx, then to tensorflow, then to tensorflow lite, and quantized from float32 to int8 using the a process detailed below. Included are the model's configuration (as detailed by the conditional lane detection repository format), the tflite model, as well as the conversion scripts and associated conda environment details.

The following process was used to get the tflite mode:
1. Download and setup repository for [conditional lane detection](https://github.com/aliyun/conditional-lane-detection).
2. Trained a model using pytorch framework with [culane dataset format](https://www.kaggle.com/datasets/greatgamedota/culane).
3. Converted the saved .pth file to .onnx using the conditional-lane-detection/tools/pytorch2onnx.py script provided. Command was:
`python tools/pytorch2onnx.py configs/lane_detection_small_train.py lane-detection-model.pth --out lane-detection-model.onnx`
4. Convert the saved .onnx file using the lane_detection_conversion_script to .tflite, which follows Coral documentation on requirements and configurations options for conversion and quantization for float32 to int8
5. Use edgetpu compiler on .tflite file:
`edgetpu_compiler lane-detection-model.tflite`

We have tried to reduce the resolution of the images in the conversion process, which was mentioned in several blog posts related to unknown edgetpu compilation failures. This did not resolve the issue. We were able to run inference on the float32 models and the int conversion model on a local development machine. Compilation has never succeeded for this model, so it has not been tested on the Coral device.

Additionally, we have checked the model operations and they all appear supported by Coral - so we are at a loss as how to proceed.

The ONNX file can be provided if necessary, but TFLite file is present (simply remove the .txt extension from it to use).

[conda_environment.txt](https://github.com/google-coral/edgetpu/files/8921809/conda_environment.txt)
[lane_detection_small_train.py.txt](https://github.com/google-coral/edgetpu/files/8921824/lane_detection_small_train.py.txt)
[lane-detection-model.tflite.txt](https://github.com/google-coral/edgetpu/files/8921851/lane-detection-model.tflite.txt)
[lane_detection_conversion_script.py.txt](https://github.com/google-coral/edgetpu/files/8921907/lane_detection_conversion_script.py.txt)

<details><summary>Click to expand!</summary> 
 
 ### Issue Type

Support, Others

### Operating System

Ubuntu

### Coral Device

USB Accelerator

### Programming Language

Python 3.7

### Relevant Log Output

```shell
Edge TPU Compiler version 16.0.384591198
Started a compilation timeout timer of 180 seconds.
Compilation child process completed within timeout period.
Compilation failed!
```
</details>

**Comment by hjonnala** [comment]
The model is failing to compile since the inference test for lane-detetction-model.tflite itself failing with [tflite_runtime-2.5.0.post1](https://github.com/hjonnala/snippets/blob/main/coral_inference.py).

https://github.com/hjonnala/snippets/blob/main/coral_inference.py

**Comment by ghost** [comment]
@hjonnala I had ran this script prior as well, and receive a `Aborted (core dumped)` message is all. Is there any other information that this can provide me? It's a little unclear what is causing the failure still.

Is there a way to output what is causing the core dump, or perform more verbose logging here?

**Comment by hjonnala** [comment]
Please check with tensorflow/tflite team regarding `Aborted (core dumped) `issue. https://github.com/tensorflow/tflite-support/issues

**Comment by ghost** [comment]
I will do that. Thanks!
